cfme/tests/automate/test_service_dialog.py

```diff
@@ -482,3 +485,129 @@ def test_automate_methods_from_dynamic_dialog_should_run_as_per_designed(
         for ele_name in [ele_label, "label2", "label3"]:
             # Checking if automate method is not triggered after updating values of dialog widgets
             view.fields(ele_name).input.fill(fauxfactory.gen_alphanumeric())
+
+
+@pytest.fixture(scope="module")
+def custom_button(appliance):
+    """This fixture creates custom button for user object"""
+    def button(dialog):
+        collection = appliance.collections.button_groups
+
+        # Create custom button group
+        button_grp = collection.create(
+            text=fauxfactory.gen_alphanumeric(),
+            hover=fauxfactory.gen_alphanumeric(),
+            type=getattr(collection, "USER"),
+        )
+
+        # Create custom button
+        button = button_grp.buttons.create(
+            text=fauxfactory.gen_alphanumeric(),
+            hover=fauxfactory.gen_alphanumeric(),
+            dialog=dialog,
+            system="Request",
+            request="InspectMe",
+        )
+        return button_grp, button
+
+    return button
+
```

Comment:
Please add buttongroup.delete_if_exists and button.delete_if_exists after fixture yield

Reply:
Yes. I agree @sshveta 
- Actually, I am adding this in [test case itself](https://github.com/ManageIQ/integration_tests/pull/9622/files#diff-8f6f36b5ece4198adbdac05597a7dc76R581). Because I tried adding in fixture. It tries to delete dialog first. So that dialog deletion gets failed because it is connected with custom button. 
- We need to import dialog before creating custom button as it is required while creating custom button